pkg/auth/handler/webapp/authflowv2/settings_biometric.go

```diff
@@ -90,7 +98,11 @@ func (h *AuthflowV2SettingsBiometricHandler) ServeHTTP(w http.ResponseWriter, r
 	defer ctrl.ServeWithoutDBTx()
 
 	ctrl.Get(func() error {
-		data, err := h.GetData(r, w)
+		var data map[string]interface{}
+		err := h.Database.WithTx(func() error {
+			data, err = h.GetData(r, w)
+			return err
+		})
```

Comment:
I suggest being consistant, and use WithTx here, instead of in GetData. Please change all other occurrences.